ios/Sources/Styleguide/Resources/ja.lproj/Localizable.strings

```diff
@@ -9,3 +9,9 @@
 
 // MARK: About
 "AboutScreen.Title" = "About";
+
+// MARK: Setting
+"SettingScreen.Title" = "Setting";
+
```

Comment:
I think this line is unnecessary, please remove 🙏

Reply:
I use this [here](https://github.com/fummicc1/conference-app-2021/blob/ed1b914de06c8fe3aaba360a6ce3c2bff4f484d9/ios/Sources/SettingFeature/SettingScreen.swift#L37)

Reply:
I meant the blank new line in line 15 🙇

Reply:
I see!!